### ISSUE TITLE
`revive` still angry with us

### ISSUE THREAD

**Issue by amery** [darvaza-proxy/darvaza#36]
I'm a cheat, but I really don't want to keep carrying this [change](https://github.com/darvaza-proxy/darvaza/commit/8967f5d2540e34f9b646d24f4455e2be59f1ab87)  forward :p

```diff
--- a/Makefile
+++ b/Makefile
@@ -9,7 +9,7 @@ GOPATH ?= $(shell $(GO) env GOPATH)
 GOBIN ?= $(GOPATH)/bin

 REVIVE ?= $(GOBIN)/revive
-REVIVE_RUN_ARGS ?= -formatter friendly -set_exit_status
+REVIVE_RUN_ARGS ?= -formatter friendly
 REVIVE_INSTALL_URL ?= github.com/mgechev/revive

 V = 0
```

we've done a pretty decent job recently but there is still more to do

```
▶ tidy: shared
  ⚠  https://revive.run/r#package-comments  should have a package comment  
  tls/server/server.go:1:1

  ⚠  https://revive.run/r#exported  exported type Proxy should have comment or be unexported  
  tls/server/server.go:26:6

  ⚠  https://revive.run/r#exported  exported method ProxyConfig.New should have comment or be unexported  
  tls/server/server.go:71:1

  ⚠  https://revive.run/r#exported  exported method Proxy.Run should have comment or be unexported  
  tls/server/server.go:91:1

  ⚠  https://revive.run/r#exported  exported method Proxy.Reload should have comment or be unexported  
  tls/server/server.go:132:1

  ⚠  https://revive.run/r#exported  exported method Proxy.TLSHandler should have comment or be unexported  
  tls/server/server.go:136:1

  ⚠  https://revive.run/r#exported  exported method Proxy.Cancel should have comment or be unexported  
  tls/server/server.go:140:1

⚠ 7 problems (0 errors, 7 warnings)

▶ tidy: root
  ⚠  https://revive.run/r#package-comments  should have a package comment  
  cmd/darvaza/config.go:1:1

  ⚠  https://revive.run/r#exported  exported const CmdName should have comment (or a comment on this block) or be unexported  
  cmd/darvaza/main.go:10:2

  ⚠  https://revive.run/r#exported  exported type Config should have comment or be unexported  
  cmd/darvaza/config.go:15:6

  ⚠  https://revive.run/r#exported  exported method Config.SetDefaults should have comment or be unexported  
  cmd/darvaza/config.go:19:1

  ⚠  https://revive.run/r#exported  exported function NewConfig should have comment or be unexported  
  cmd/darvaza/config.go:29:1

  ⚠  https://revive.run/r#exported  exported method Config.ReadInFile should have comment or be unexported  
  cmd/darvaza/config.go:39:1

  ⚠  https://revive.run/r#exported  exported method Config.WriteTo should have comment or be unexported  
  cmd/darvaza/config.go:43:1

⚠ 7 problems (0 errors, 7 warnings)
```

**Comment by karasz** [comment]
If you remove the set exit status and make will not choke on the revive errors then we will never fix them. I think revive can force us to write better code.

**Comment by amery** [comment]
fixed by d1b724c89401759f540dacb82589847e0fa20419
